spec/models/transaction_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Transaction, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :credit_card_number }
+    # it { should validate_presence_of :credit_card_expiration_date }
```

Comment:
I'm skipping validation for credit card expiration dates for the time being since none of the transactions from the CSV file seem to have one.

Reply:
👍